app/retail/templates/shared/footer_scripts.html

```diff
@@ -23,9 +23,9 @@
 </script>
 <script src="{% url 'javascript-catalog' %}"></script>
 <script src="{% static "v2/js/lib/jquery.js" %}"></script>
-{% if user.is_authenticated and env == 'prod' %}
+{% if env == 'prod' %}
   <script src="https://cdn.jsdelivr.net/npm/vue"></script>
-{% elif user.is_authenticated %}
+{% else %}
   <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
```

Comment:
random question - should we pin this version or bring it into our repo as we do with other libs?